src/main.ts

```
@@ -2600,12 +2615,47 @@ ipcMain.handle('scan-closed-vscode-sessions', async (_event, activeSessionIds: s
   return scanClosedVSCodeSessions(new Set(activeSessionIds), vsCodeIndex);
 });
 
-ipcMain.on('open-claude-session', async (_event, sessionId: string, projectPath: string, isActive: boolean, activePid?: number, customTitle?: string) => {
+ipcMain.on('open-claude-session', async (_event, sessionId: string, projectPath: string, isActive: boolean, activePid?: number, customTitle?: string, accountLabel?: string) => {
```

ESLint

[error] Replace `'open-claude-session',·async·(_event,·sessionId:·string,·projectPath:·string,·isActive:·boolean,·activePid?:·number,·customTitle?:·string,·accountLabel?:·string` with `⏎··'open-claude-session',⏎··async·(⏎····_event,⏎····sessionId:·string,⏎····projectPath:·string,⏎····isActive:·boolean,⏎····activePid?:·number,⏎····customTitle?:·string,⏎····accountLabel?:·string,⏎··`

(prettier/prettier)